src/main/bin/resolveOptions.ts

```diff
@@ -1,33 +1,22 @@
 import { lstatSync } from 'fs'
 
-import { DEFAULT_OPTIONS } from '../options'
+import {
+    DEFAULT_APACHE_LIB,
+    DEFAULT_OPTIONS,
```

Comment:
DEFAULT_OPTIONS in options is different from the one in defaults. It is also better to have a single source for it.